cephalopod/utils.py

```diff
@@ -7,6 +7,9 @@
 import pkg_resources
 from flask import current_app
 from flask.helpers import get_root_path
+from flask_mail import Message
+
+from cephalopod.core import mail
```

Comment:
might be safer to move this import inside the function. i wouldn't expect utils to import core